docker/docker-bootstrap.sh

```diff
@@ -47,5 +47,5 @@ elif [[ "${1}" == "app" ]]; then
   flask run -p 8088 --with-threads --reload --debugger --host=0.0.0.0
 elif [[ "${1}" == "app-gunicorn" ]]; then
   echo "Starting web app..."
-  /app/docker/docker-entrypoint.sh
+  /usr/bin/run-server.sh
```

Comment:
Having trouble starting docker composer due to this fix:

```
/app/docker/docker-bootstrap.sh: line 50: /usr/bin/run-server.sh: No such file or directory
```

Reply:
Try running `docker-compose pull` first

Reply:
> 

macOS 
 /usr/bin/run-server.sh   ->  sh /app/docker/run-server.sh